### PR TITLE
Fix codying style

### PR DESCRIPTION
**Pull request by Stronkness**
Used pylint and shellcheck to check for syntax error or bugs 

Solves #8

### CODE REVIEW

File: gitapper.sh [hunk 1]
```diff
@@ -1,40 +1,40 @@
 #!/usr/bin/env bash
 set -e
 # Debug mode
-[[ -v debug ]] && set -x
+#[[ -v debug ]] && set -x
```
Comment (Mte90) [inline gitapper.sh:4]: Why did you commented this?

**Comment by Mte90** [comment]
Did you tested the code after those changes?

File: gitapper.sh (hunk 1 above)
Comment (Stronkness) [reply gitapper.sh:4]: In the original file the debug code line was commented out. I forgot to comment it out in my commit hence why i did like this

**Comment by Stronkness** [comment]
> Did you tested the code after those changes?

I'm sorry. It looks like i forgot to add this comment in my PR.

Sadly my Linux machine broke down and I can't use it right now. I was supposed to ask you if you could test if it still works. Otherwise I have to wait a couple of days before I can properly test it. You don't have to test it if you don't have the time. We can wait a few days otherwise

File: gitapper.sh (hunk 1 above)
Comment (Stronkness) [reply gitapper.sh:4]: See this commit https://github.com/Mte90/gitapper/pull/10/commits/d7fa7729d6208d57fd73a813c8fe94a9ee966aac

Comment (Mte90) [reply gitapper.sh:4]: Ok but this line now is uncommented so it should stay in that way

**Comment by Mte90** [comment]
Because I am not sure that wrapping all the bash variables let the code to split the various parameters shared to the script so require some testing.

**Comment by Stronkness** [comment]
Okey, i can try in a few days if you dont want to do it :)

However, this is what the linters told me to do

**Comment by Stronkness** [comment]
Hi again! My linux machine works again and i have tested every functionality stated in the readme file. And yes, everything works as intended.

**Comment by Stronkness** [comment]
I also thought on if maybe some "echoes" would be good for the user so they know if it failed or not. However, that might be another issue

**Comment by Mte90** [comment]
I tested that patch but seems that commands like `git add` that use forgit doesn't work anymore...

**Comment by Stronkness** [comment]
Thats very strange. I will take a look at it

**Comment by Stronkness** [comment]
This is strange, i just added commits and pushed them and everything works fine

**Comment by Mte90** [comment]
Did you tried `git checkout` as example?

**Comment by Stronkness** [comment]
Yes that works

**Comment by Mte90** [comment]
Did you tried just `git checkout` without any parameters?

**Comment by Stronkness** [comment]
I can ty that. Which terminal lines doesnt seem to work at your end? So i can try them all out

**Comment by Mte90** [comment]
It was my fault, I should remember to do tests when I have anything to do not while I am doing other things...

**Comment by Stronkness** [comment]
No worries. Glad i could help!

**Pull request merged**